app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: 'stella.k.bonnie@gmail.com'
```

Comment:
I am unsure if we have to use an actual email here.

Reply:
I think we can send it from any email, i.e. `do_not_reply@...com`